app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html

```diff
@@ -9,25 +9,25 @@
         <md-card-content>
             <form name="addBeneficiaryForm" ng-submit="addBeneficiaryForm.$valid && vm.submit()">
                 <md-input-container class="md-block md-icon-float">
-                    <label>{{ 'label.input.accountType' | translate }} <span class="required">*</span></label>
```

Comment:
@botraunak Are these fields not required ? Why we removed this ?

Reply:
I just removed the asterik. on the latest material the select already has an asterik.